converting submit-app guide to single-page (#2858)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/submit-app/main/index.md

### Instance URL

For each protocol, enter the **Okta instance URL** for your integration in the first field.

To get your Okta instance URL in your development org:

1. In the Okta Admin Console, go to **Applications** > **Applications** to see all the integrations in your org.
1. Click the name of the app integration that you are going to submit.
1. On the settings page, confirm that the settings match what you want as the global defaults for all customers.
1. In your browser, click in the address bar showing the current URL and copy it to your clipboard. This is the Okta instance URL for your integration.
1. Back in the OIN Manager, paste that URL into your submission.

<!--
1. Click the **General** tab.
1. Go to the **App Embed Link** section and copy the text in the Embed Link field:
   ![App Embed Link](/img/oin/isv-portal_app-embed-link.png "App Embed Link GUI in the Application settings")
1. Paste that value into your submission.
-->

### Protocol-specific settings

Each of the supported protocols has different configuration settings for the remainder of the submission.

> <StackSelector snippet="submit" noSelector />

Reviewer: ```suggestion
<StackSnippet snippet="submit" />
```

Similarly, when you enable a new capability in your application (for example, adding SCIM provisioning onto an existing published SAML application), you don't need to create an entirely new submission. You can update your existing submission to enable and specify the settings for that protocol, then submit the updated integration.

1. Sign on to the OIN Manager using the credentials for your original dev org.
    >**Note:** You must submit the updated integration using the same dev org that was used to make the original submission, otherwise the Okta OIN team rejects the update.
1. The published integration appears on your integrations page. Click **Update**.

    This creates a new instance of your integration where you can safely change any of the parameters. Your existing integration remains in the OIN catalog and keeps the previous settings until this new version is published.
1. Update any of the parameters for your existing protocols, or add a new protocol depending on your needs.

>     If you need to leave your in-progress submission at any point, you can return to it through the OIN Manager. When you sign on again, you will see the published version and your in-progress version. Click **Edit** on the in-progress version to pick up where you left off.

Reviewer: ```suggestion
    If you need to leave your in-progress submission at any point, you can return to it through the OIN Manager. When you sign on again, the published version and your in-progress version appears. Click **Edit** on the in-progress version to pick up where you left off.
```

> 1. When you have completed the updates, or have filled the new protocol information so that indicator shows 100% complete, you can click **Submit for Review**.

Reviewer: ```suggestion
1. When you complete the updates or fill in the new protocol information so that the indicator shows 100% complete, you can click **Submit for Review**.
```

    At this point, the Okta OIN team is notified and your submission undergoes the same process flow as the original submission.

>     After the new version of the integration has reached the **Publish** stage and been published by Okta, the new version replaces the old one in the OIN catalog.

Reviewer: ```suggestion
    After the new version of the integration reaches the **Publish** stage and is published by Okta, the new version replaces the old one in the OIN catalog.
```
